dasharo-compatibility/coreboot-fan-control.robot

```diff
@@ -31,7 +31,7 @@ CFN001.001 CPU temperature and fan speed can be read (Debian 11.02)
     Skip If    not ${COREBOOT_FAN_CONTROL_SUPPORT}    CFN001.001 not supported
     Skip If    not ${TESTS_IN_DEBIAN_SUPPORT}    CFN001.001 not supported
     Power On
-    Boot From USB
+    Boot System Or From Connected Disk    ${USB_LIVE}
```

Comment:
@Psotas What is USB_LIVE?

It is only defined for a handful of platforms

```
./protectli-v1210.robot:${USB_LIVE}=                                        USB
./protectli-v1410.robot:${USB_LIVE}=                                        USB
./protectli-v1610.robot:${USB_LIVE}=                                        USB
./protectli-vp2410.robot:${USB_LIVE}=                                        USB
./protectli-vp2410.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
./protectli-vp2420.robot:${USB_LIVE}=                                        USB
./protectli-vp2420.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
./protectli-vp4630.robot:${USB_LIVE}=                                        USB SanDisk 3.2Gen1
./protectli-vp4630.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
./protectli-vp4650.robot:${USB_LIVE}=                                        USB SanDisk 3.2Gen1
./protectli-vp4650.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
./protectli-vp4670.robot:${USB_LIVE}=                                        USB SanDisk 3.2Gen1
./protectli-vp4670.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
./raptor-cs_talos2.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
./raptor-cs_talos2.robot:${USB_LIVE}=                                        USB: sda2
./qemu.robot:${USB_LIVE}=                                        USB SanDisk 3.2Gen1
./qemu.robot:@{ATTACHED_USB}=                                    ${USB_LIVE}
```

Reply:
https://gitlab.com/3mdeb/rte/open-firmware-rte/-/blob/master/keywords.robot?ref_type=heads#L1941

I tried to recreate the operation of keywords.
I didn't go too deep into it. I focused only on removing `Boot From USB`.
But it's probably better to take a closer look at these tests, what they were supposed to do and whether they really work.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/issues/143